src/app/data-planes/views/DataPlaneSummaryView.vue

```diff
@@ -76,7 +76,6 @@
                       >
                         <KTooltip
                           class="reason-tooltip"
-                          placement="bottomEnd"
```

Comment:
This was removed purposefully, in the upgrade the string was changed from `bottomEnd` to `bottom-end`, but seeing this is the only place where we use `placement`, I tried removing it to see if it made any difference, and it didn't.

FloatingUI probably has better positioning, and I'd rather not have to define/hardcode positioning on things anyway.